webpack/base.js

```diff
@@ -5,11 +5,11 @@ module.exports = () => {
   return {
     entry: {
       "login": [
-        "./src/apps/loyalty/containers/login/browser",
+        "babel-polyfill", "./src/apps/loyalty/containers/login/browser",
       ],
       "inquiries": [
-        "./src/apps/loyalty/containers/inquiries/browser",
-      ]
+        "babel-polyfill", "./src/apps/loyalty/containers/inquiries/browser",
```

Comment:
I'm not actually sure whether webpack does that or not. I'll double-check that as well 👍